assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceItem.js

```diff
@@ -108,6 +114,25 @@ export default function AudienceItem( {
 
 	const id = `audience-selection-checkbox-${ slug }`;
 
+	function ItemBadge() {
+		if ( temporarilyHidden ) {
+			return (
+				<BadgeWithTooltip
+					className="googlesitekit-audience-segmentation-temporary-hidden-badge"
+					label={ __( 'Temporarily hidden', 'google-site-kit' ) }
+					tooltipTitle={ __(
+						'Site Kit is collecting data for this group. once data is available the this group will be added to your dashboard.',
```

Comment:
While this text does match the AC, it feels grammatically incorrect. I've raised it in the issue [here](https://github.com/google/site-kit-wp/issues/9096#issuecomment-2362344972). Let's see what Tom thinks.

Reply:
Thanks @nfmohit, good spot. I've [replied](https://github.com/google/site-kit-wp/issues/9096#issuecomment-2363433595) on the issue and amended the AC/IB.